packages/worker/src/worker.ts

```diff
@@ -413,6 +417,7 @@ export class Worker<T extends WorkerSpec = DefaultWorkerSpec> {
   protected readonly numInFlightActivationsSubject = new BehaviorSubject<number>(0);
   protected readonly numInFlightActivitiesSubject = new BehaviorSubject<number>(0);
   protected readonly numRunningWorkflowInstancesSubject = new BehaviorSubject<number>(0);
+  private readonly runIdsToSpanContext = new Map<string, SpanContext>();
```

Comment:
Looks like we never delete from this map, is there a way around it?

Reply:
On eviction probably

Reply:
Fixed this